src/sidebar/cross-origin-rpc.js

```diff
@@ -38,28 +53,25 @@ function start(store, settings, $window) {
 
   /** Return a JSON-RPC response to the given JSON-RPC request object. */
   function jsonRpcResponse(request) {
-    // The set of methods that clients can call.
-    let methods = {
-      searchUris: store.searchUris,
-    };
-
-    let method = methods[request.method];
-
-    let response = {
-      jsonrpc: '2.0',
-      id: request.id,
-    };
+    // Find the mapped method requested
+    const method = methods[request.method];
 
-    if (method) {
-      response.result = method();
-    } else {
-      response.error = {
-        code: -32601,
-        message: 'Method not found',
+    // Return an error response if the method name is invalid.
```

Comment:
Perhaps be more specific than "invalid" and say "not registered as an allowed method" or something?